### DISCUSSION TITLE
Icons & Logos

### DISCUSSION THREAD

**Discussion by stratofax** [Security-Best-Practices/SBP-web#6] [Ideas]
You might want to grab the logos of some of your clients. Also, consider some icons, like these [Font Awesome Icons](https://fontawesome.com/v4/icons/)

**Comment by stratofax** [comment]
I like these icons for your site: [Lucide](https://lucide.dev/)

**Comment by SecurityBestPractices** [comment]
Wagner Law Group - transparent
![WLG Logo transparent](https://github.com/user-attachments/assets/f16e5bce-5534-4793-83aa-1c0745922fef)

**Reply by SecurityBestPractices** [reply]
White background.
![SMF WLG Logo](https://github.com/user-attachments/assets/df269bf4-8ea4-4c13-903a-dbd58100bf19)

**Comment by SecurityBestPractices** [comment]
Another transparent (?) or maybe it's a white background.
![WLG Logo transparent](https://github.com/user-attachments/assets/dc78373c-469b-4960-9fc4-e7c0c401b6d6)

**Comment by SecurityBestPractices** [comment]
BNHC
![BNHC-logo](https://github.com/user-attachments/assets/cf007c03-7de8-4d9c-91d0-b57b0d252d9e)

**Comment by SecurityBestPractices** [comment]
Bentley Publishing
![BentleyPublishers](https://github.com/user-attachments/assets/ee15ad23-9361-432a-bcca-00ca8bac066b)

**Comment by SecurityBestPractices** [comment]
National DCP
![NationalDCP](https://github.com/user-attachments/assets/eb265b34-08b1-41d6-b3fd-7b6075e69b96)

**Comment by SecurityBestPractices** [comment]
Allied Domecq (owned Dunkin Brands at the time). Maybe I should put the Dunkin Logo instead?
![Allied_20Domecq_20logo](https://github.com/user-attachments/assets/8ed781b0-8767-45c7-bb45-16dab5f6989a)

**Reply by SecurityBestPractices** [reply]
![Dunkin'+Brands+Logo_69c379da-4bea-4dbc-9e6e-a81f0bdaf4b2-prv](https://github.com/user-attachments/assets/5938328d-0079-401e-b524-e576935bc2be)

**Reply by SecurityBestPractices** [reply]
Cropped:
![DunkinBrand](https://github.com/user-attachments/assets/352b7c3a-e0a3-4d17-9454-60b6ab344815)

**Comment by SecurityBestPractices** [comment]
Svenska Handelsbanken
![Svenska-Handelsbanken-AB-logo](https://github.com/user-attachments/assets/4b31aeee-4a01-4f53-bd11-6abe0e5922de)

**Reply by SecurityBestPractices** [reply]
Cropped:
![image](https://github.com/user-attachments/assets/fe5d98f9-f2d8-4867-abaa-dcf61a9a446a)

**Comment by SecurityBestPractices** [comment]
ABN Amro:
![ABN_AMRO_logo](https://github.com/user-attachments/assets/a2fc220a-43b9-468d-9569-cb67a6f2ecec)

**Comment by SecurityBestPractices** [comment]
GV20 Therapeutics https://www.gv20tx.com/
![GV20_Logo](https://github.com/user-attachments/assets/1f6182da-c2a0-4148-b456-0b42668610e5)

**Comment by SecurityBestPractices** [comment]
Fragomen, Del Rey, Bernsen & Loewy, LLP
![fragomen-logo-1621530729](https://github.com/user-attachments/assets/30c43592-da81-4282-9c65-3f04418c158d)

**Comment by stratofax** [comment]
@SecurityBestPractices Great client list! I like to convert all the logos to monochrome so they don't fight with your web palette, plus they need to be roughly the same size. You can see what I mean in the Clients section of the [Cadent Home page](https://cadent.com/)

**Comment by SecurityBestPractices** [comment]
![abn-amro-logo-white](https://github.com/user-attachments/assets/56567359-dcce-4091-9212-0ca7a792f5dc)

**Reply by stratofax** [reply]
Looks good!

**Comment by SecurityBestPractices** [comment]
![bentley-publishers-logo-white](https://github.com/user-attachments/assets/51530e21-ca9a-4f39-95cf-563f349694f5)

**Reply by stratofax** [reply]
Perfect

**Comment by SecurityBestPractices** [comment]
![bnhc-logo-white](https://github.com/user-attachments/assets/71d5e93f-3ba0-4f2a-aa8e-b544220d7d75)

**Reply by stratofax** [reply]
Needs to be white

**Reply by SecurityBestPractices** [reply]
I thought this was white? I stole it from your website. Anyway, trying some things now. It's weird that if I import the PNG into GIMP, it's all grainy.

**Reply by SecurityBestPractices** [reply]
![bnhc-logo-white](https://github.com/user-attachments/assets/ae0caa83-92b1-4d91-a8f7-8b4d0f174c6f)

**Reply by SecurityBestPractices** [reply]
Got it. BTW, the trick to fixing the graininess was the 3rd of 5 recommendations Perplexity gave me:
Color Mode: The image might be in indexed color mode, which can sometimes cause a grainy appearance. To fix this, try changing the color mode to RGB by going to Image > Mode > RGB in GIMP

**Reply by stratofax** [reply]
Yes, I'm pretty sure I indexed the colors to reduce the file size.

**Comment by SecurityBestPractices** [comment]
![dunkin-donuts-logo-white](https://github.com/user-attachments/assets/d56dfab9-7003-4f63-980e-2e9e7a3a0c16)

**Reply by stratofax** [reply]
Mmm, coffee! Well done

**Comment by SecurityBestPractices** [comment]
![fragomen-logo-white](https://github.com/user-attachments/assets/82b0a6de-4e68-47ef-baf7-4f5e3cfbb30c)

**Reply by stratofax** [reply]
Nailed it

**Comment by SecurityBestPractices** [comment]
![gv20-logo](https://github.com/user-attachments/assets/8821ecb1-3618-44f3-8348-49082bdb5284)

**Reply by stratofax** [reply]
Their logo is too complicated, but this looks good

**Comment by SecurityBestPractices** [comment]
![Handelsbanken-logo-white](https://github.com/user-attachments/assets/94cd3fb7-6d85-45a9-b6e1-1765b23bc8b5)

**Reply by stratofax** [reply]
Score!

**Comment by SecurityBestPractices** [comment]
![wagner-law-group-logo-white](https://github.com/user-attachments/assets/0dfa7772-c889-4e32-8c19-289adcdbdfb3)

**Reply by stratofax** [reply]
Good job with a bad logo

**Comment by SecurityBestPractices** [comment]
![ndcp-logo-white](https://github.com/user-attachments/assets/2a08d572-ea9d-4b0e-86f6-cb6e87cbe465)

**Reply by stratofax** [reply]
Nice!

**Comment by SecurityBestPractices** [comment]
Weird how it got grainy, I’ll try switching it back to indexed to see if it messes it up.
The biggest logo is Fragomen, at 171KB, I’ll check that one also.

**Comment by SecurityBestPractices** [comment]
Oops I was looking at the wrong file, Fragomen is only 11kb.


From: Keith W Salustro
Sent: Thursday, January 2, 2025 12:43 PM
To: Security-Best-Practices/SBP-web ***@***.***>; Security-Best-Practices/SBP-web ***@***.***>
Subject: RE: [Security-Best-Practices/SBP-web] Icons & Logos (Discussion #6)

Weird how it got grainy, I’ll try switching it back to indexed to see if it messes it up.
The biggest logo is Fragomen, at 171KB, I’ll check that one also.
